src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -166,17 +178,17 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
 
         if (typeof formTemplate.layout !== 'undefined'
             && Array.isArray(formTemplate.layout.sections)) {
-
             // add record data to the template fields
-            if (isNotEmpty(fieldMappings) && isNotEmpty(this.donorRecord)) {
-                let sectionsWithValues = setRecordValuesOnTemplate(formTemplate.layout.sections,
+
+             if (isNotEmpty(fieldMappings) && isNotEmpty(this.donorRecord)) {
```

Comment:
This line is just *one* space too far indented, I think!

Reply:
You're right! Missed it.